sqladmin/application.py

```diff
@@ -156,6 +157,8 @@ def __init__(
         base_url: str = "/admin",
         title: str = "Admin",
         logo_url: str = None,
+        middleware: Sequence[Middleware] = None,
```

Comment:
I know that Starlette expects `middleware` keyword, but since this is a sequence, I think it's better to switch this to `middlewares` on the SQLAdmin side.
```suggestion
        middlewares: Sequence[Middleware] = None,
```